otherlibs/unix/unix.mli

```diff
@@ -390,6 +390,12 @@ val read_bigarray :
 (** Same as {!read}, but read the data into a bigarray.
     @since 5.2 *)
 
+val nonblock_read : file_descr -> bytes -> int -> int -> int
+(** Same as {!read}, but does not release the global lock nor copy the
+    bytes read. It is only safe to use with non blocking file descriptor,
```

Comment:
It is not completely clear what does "safe" means here. What about "It should only be used with ...".